src/commands/deploy.rs

```diff
@@ -153,11 +153,12 @@ impl DeployCommand {
             name.clone(),
             bindle_id.version_string()
         );
-        if is_http_config {
+        if let Ok(http_config) = HttpTriggerConfiguration::try_from(cfg.info.trigger.clone()) {
             let channel = Client::get_channel_by_id(&hippo_client, &channel_id)
                 .await
                 .context("Problem getting channel by id")?;
             println!("Application is running at {}", channel.domain);
```

Comment:
What do you think about removing this line entirely?

Reply:
I don't think we can.  We need to type the trigger/config so that we can access the `.base` field.

Reply:
Perhaps move it into an `else` clause of this `if`?

Reply:
Oh, duh, I misread which line Michelle was referring to!